src/main/java/org/zalando/springframework/web/logging/DefaultHttpLogger.java

```diff
@@ -28,7 +28,7 @@
 import java.util.List;
 import java.util.Map;
 
-public class DefaultHttpLogger implements HttpLogger {
+public final class DefaultHttpLogger implements HttpLogger {
```

Comment:
In that case they should open a pull request and give a good reason for the change :)